pkg/controller/render/render_controller.go

```diff
@@ -496,6 +496,7 @@ func (ctrl *Controller) syncGeneratedMachineConfig(pool *mcfgv1.MachineConfigPoo
 	if apierrors.IsNotFound(err) {
 		_, err = ctrl.client.MachineconfigurationV1().MachineConfigs().Create(context.TODO(), generated, metav1.CreateOptions{})
 		glog.V(2).Infof("Generated machineconfig %s from %d configs: %s", generated.Name, len(source), source)
+		ctrl.eventRecorder.Eventf(pool, corev1.EventTypeNormal, "RenderedConfigGenerated", "%s successfully generated", generated.Name)
```

Comment:
Nit: Not a new issue, but we'll log this even if an error occurs which could be confusing, maybe hoist up the `if err != nil` check to be before this?

Reply:
hmm good call!

Reply:
wait i think i need to keep the old one _and_ add the new one in the if?

Reply:
Yeah we need both, we want to return an error if create fails, or if there was some other error.

Reply:
done!